test/integration/tests/loadexternal.sh

```diff
@@ -92,4 +93,22 @@ tpm2_create -Q -C $Handle_parent -g $alg_create_obj -G $alg_create_key -u $file_
 
 tpm2_loadexternal -Q -u $file_loadexternal_key_pub
 
+# Test loading an OSSL generated private key with a password
+openssl genrsa -out private.pem 2048
+openssl rsa -in private.pem -out public.pem -outform PEM -pubout
+
+echo "hello world" > plain.txt
+openssl rsautl -encrypt -inkey public.pem -pubin -in plain.txt -out plain.rsa.enc
+
+tpm2_loadexternal -a o -p foo -r private.pem -o key.ctx
+
+#
+# This command is failing with:
+# ERROR: Tss2_Sys_RSA_Decrypt(0x12F) - tpm:error(2.0): authValue or authPolicy is not
+# available forselected entity
```

Comment:
I *think* I know what is missing. I forgot to initialize the auth structure with the handle for the password session.